handlers/get_token_meta_handler.py

```diff
@@ -0,0 +1,28 @@
+import json
+from typing import Union
+from usecases.get_token_metadata import GetTokenMetadata
+
+from aws_lambda_context import LambdaContext
+from utils.wrappers.aws.api_gateway import ApiGateway, ApiGatewayEvent
+
+
+@ApiGateway()
+def handle(
+    event: ApiGatewayEvent,
+    _context: LambdaContext,
+    get_token_metadata: Union[GetTokenMetadata, None] = None
+) -> dict:
+
+    get_token_metadata = get_token_metadata or GetTokenMetadata()
+    response = get_token_metadata.get(event.path['hash'])
+
+    if response is None:
+        raise Exception('not_found')
```

Comment:
I think we should return 404 here, it's better for the client to handle tokens that don't have metadata

Reply:
raising `'not_found'` string will lead to a 404 response. I think we can improve it with constants